packages/botonic-react/src/components/multichannel/multichannel.jsx

```diff
@@ -8,13 +8,29 @@ import { MultichannelCarousel } from './multichannel-carousel'
 import { MultichannelContext } from './multichannel-context'
 import { MultichannelReply } from './multichannel-reply'
 import { MultichannelText } from './multichannel-text'
-import { isWhatsapp, MULTICHANNEL_WHATSAPP_PROPS } from './multichannel-utils'
+import {
+  isFacebook,
+  isWhatsapp,
+  MULTICHANNEL_WHATSAPP_PROPS,
+} from './multichannel-utils'
 
 export const Multichannel = props => {
   const requestContext = useContext(RequestContext)
 
-  if (!isWhatsapp(requestContext)) {
+  if (!isWhatsapp(requestContext) && !isFacebook(requestContext)) {
     return props.children
+  } else if (isFacebook(requestContext)) {
+    const newChildren = deepMapWithIndex(props.children, child => {
+      if (child && child.type && child.type.name === 'Text') {
+        return (
+          <MultichannelText key={child.key}>
```

Comment:
what happens with Text props? See whatsapp example below.
This function is too long.
maybe you can do st like:
```
function getConverters() {
if (isFacebook(requestContext)) { 
return {
'Text': convertText,
}
if (isWhatsapp(requestContext)) { 
return {
'Text': convertText,
'Carousel': convertCarousel
...
}
}
const converters = getConverters()
if converters[child.type.name]) {
return converters[child.type.name](child)
})
else
   return child
```

call me if you have doubts